### PR TITLE
Final numbers

### PR DESCRIPTION
**Pull request by martinthomson**
For those drafts that are in AUTH48.

### CODE REVIEW

**Review by nibanks — APPROVED** [review]
9000! Very cool!

**Review by ianswett — APPROVED** [review]

**Review by marten-seemann — APPROVED** [review]

File: rfc8999.md [hunk 1]
```diff
@@ -1,9 +1,11 @@
 ---
 title: "Version-Independent Properties of QUIC"
 abbrev: QUIC Invariants
-docname: draft-ietf-quic-invariants-latest
-date: {DATE}
+number: 8999
+docName: draft-ietf-quic-invariants-13
+date: 2021-05
```
Comment (seanturner) [inline rfc8999.md:6]: Should this be {DATE} or should {DATE} in draft-ietf-quic-http.md be 2021-05?

Comment (MikeBishop) [reply rfc8999.md:6]: {DATE} will auto-populate whenever the doc builds, though.  It'll be a fixed value after this, but HTTP doesn't have them yet.

Comment (seanturner) [reply rfc8999.md:6]: Ah that's right!

**Review by seanturner — APPROVED** [review]

**Review by janaiyengar — APPROVED** [review]
This PR makes me happy.

**Pull request merged**